SECURITY.md

```diff
@@ -0,0 +1,212 @@
+  destroying data, and/or harming user experience;
+- Use only the [Official Channels](#reporting-a-vulnerability) to discuss
+  vulnerability information with us;
+- Keep the details of any discovered vulnerabilities confidential until they are
+  fixed, according to the Disclosure Policy;
+- Perform testing only on in-scope systems, and respect systems and activities
+  which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of
+  data you access to the minimum required for effectively demonstrating a Proof
+  of Concept; and cease testing and submit a report immediately if you encounter
+  any user data during testing, such as Personally Identifiable Information
+  (PII), Personal Healthcare Information (PHI), credit card data, or proprietary
+  information;
+- You should only interact with test accounts and Knative instances you own or
+  with explicit permission from the owner or account holder; and
+- Do not engage in extortion.
+
+## Safe Harbor
+
+When conducting vulnerability research according to this policy, we consider
+this research to be:
+
+- Authorized in accordance with the Computer Fraud and Abuse Act (CFAA) (and/or
+  similar state laws), and we will not initiate or support legal action against
+  you for accidental, good faith violations of this policy;
+- Exempt from the Digital Millennium Copyright Act (DMCA), and we will not bring
+  a claim against you for circumvention of technology controls;
+- Exempt from restrictions in our Terms & Conditions that would interfere with
+  conducting security research, and we waive those restrictions on a limited
+  basis for work done under this policy; and
+- Lawful, helpful to the overall security of the Internet, and conducted in good
+  faith.
+
+You are expected, as always, to comply with all applicable laws.
+
+If at any time you have concerns or are uncertain whether your security research
+is consistent with this policy, please submit a report through one of our
+[Official Channels](#reporting-a-vulnerability) before going any further.
+
+## Scope and Supported Versions
+
+We (the community) support the 4 most recent minor versions of Knative. For more
+details on what this means, see our
+[release principles](https://knative.dev/community/contributing/mechanics/release-versioning-principles/).
+
+While we encourage you to report vulnerabilities in any Knative repo, at HEAD or
+the tip of any release branch (release-X.Y), we commit to patching
+vulnerabilities only at HEAD and in currently supported release branches.
+
+Repos with releases have the releases listed in the Github UI, e.g.:
+
+- [serving](https://github.com/knative/serving/releases)
+- [eventing](https://github.com/knative/eventing/releases)
+
+Repos in Knative Sandbox that have no releases listed may have a higher bar for
+accepting vulnerability reports. No support is provided for archived repos.
+
+## Reporting a Vulnerability
+
+Please email the
```

Comment:
Perhaps we should use the GitHub Security Advisory system?

Reporters contact Knative Security asking for an advisory in one or more repos. KS does so, and adds the reporter as a collaborator. The reporter then provides the rest of the details there.

Reply:
https://docs.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories

Reply:
Looks very good to me. 👍

Reply:
As I noted yesterday, I think we should only use the Security Advisory system for accepted vulnerabilities, and should simplify the workflow described in this draft.

https://github.com/knative/community/pull/258#issuecomment-777153267

WDYT of that @vaikas ?

Reply:
Just to make sure I understand:
 1. Send email to report as described
 2. Use Github Security Advisory once report from number 1 gets "accepted"

Reply:
Yup